src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts

```diff
@@ -73,7 +73,7 @@ export class CreateChildComponent extends CreateFormComponent implements OnInit,
     this.ChildrenFormArray.valueChanges.pipe(
       takeUntil(this.destroy$),
     ).subscribe((val: boolean) => {      
-      this.isEmpty = !Boolean(val[0].lastName) || !Boolean(val[0].firstName);
+      this.isEmpty = !!!(val[0].lastName) || !!!(val[0].firstName) || !!!(val[0].middleName);
```

Comment:
!val[0].lastName
!val[0].firstName
!val[0].middleName

Reply:
wrong typo? Why val is boolean but has lastName, firstName?

Reply:
I think you should simplify is to !(val[0].lastName || val[0].firstName || val[0].middleName)